core/primitives/src/stateless_validation/contract_distribution.rs

```diff
@@ -82,7 +82,7 @@ impl ChunkContractAccessesV1 {
         signer: &ValidatorSigner,
     ) -> Self {
         let inner = ChunkContractAccessesInner::new(next_chunk, contracts, main_transition);
-        let signature = signer.sign_chunk_contract_accesses(&inner);
+        let signature = signer.sign_bytes(&borsh::to_vec(&inner).unwrap());
```

Comment:
While at it, could you make all associated "inner" fields private? I remember atleast for partial witness the only reason inner was public was for the validator signer api.

Reply:
`inner` is already private for the types touched in this PR

Reply:
Aah sorry, I meant `pub struct ChunkContractAccessesInner` -> `struct ChunkContractAccessesInner`